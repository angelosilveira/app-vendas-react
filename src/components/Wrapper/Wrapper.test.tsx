import { screen } from '@testing-library/react';
import { renderWithTheme } from 'src/utils/tests/renderWithTheme';
import { describe, test, expect } from 'vitest';

import { Wrapper } from '.';

describe('<Wrapper />', () => {
  test('should render the wrapper', () => {
    const { container } = renderWithTheme(
      <Wrapper>
        <span>App vendas</span>
      </Wrapper>
    );

    expect(screen.getByText(/app vendas/i)).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
