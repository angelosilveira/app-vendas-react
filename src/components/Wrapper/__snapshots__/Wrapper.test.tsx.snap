// Vitest Snapshot v1

exports[`<Wrapper /> > should render the wrapper 1`] = `
.c0 {
  max-width: 480px;
  background-color: #fff;
  margin: 0 auto;
  height: 100%;
  width: 100%;
  position: relative;
  overflow: hidden;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<main
  class="c0"
>
  <span>
    App vendas
  </span>
</main>
`;
